src/pages/discovery/discovery.js

```diff
@@ -68,6 +69,7 @@ export default function Discovery() {
 					}
 					// ensure that sort icons in data table columns are reset since the newly received data is unsorted
 					setIsSortIconResetNeeded(true)
+					setDataLoaded(true);
```

Comment:
Add a brief comment here to explain what the purpose is of setting dataLoaded to true. Because we have a lot of new devs come through Spacelab, we benefit from code that is slightly more documented than what would normally be expected.